extensions/wikia/ARecoveryEngine/ARecoveryEngine.i18n.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+$messages = [ ];
+
+$messages["en"] = [
+	"arecovery-blocked-message-headline" => "Ad blocker interference detected!",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Ad blocker interference detected!" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "arecovery-blocked-message-headline" 12 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)